docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -284,8 +321,16 @@ This error can occur when the `/prisma/schema.prisma` file contains the incorrec
 
 If you don't enable the proper OAuth scopes, the `/api/auth` request might fail. Go to the [Dev Portal](https://devtools.bigcommerce.com) to verify the proper OAuth scopes are enabled in your app's profile. To learn more, read [Managing Apps in the Developer Portal](/api-docs/apps/guide/developer-portal#edit-technical-details). Make sure your scopes equal or exceed those listed in the [Creating an app profile](#create-an-app-profile) section.
 
+### Issue: Unable to migrate your data to another provider. 
+
+In order to manually switch the database provider, you must do the following:
```

Comment:
Change intro to: "To manually switch the database provider, complete the following steps:"